navigation/BottomTabNavigator.tsx

```diff
@@ -3,6 +3,7 @@ import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
 import { createStackNavigator } from '@react-navigation/stack';
 import * as React from 'react';
 
+import { TouchableOpacity } from 'react-native-gesture-handler';
```

Comment:
This import doesn't seem to be used anywhere, right? So we can revert the changes to this file.